src/gen-ts.ts

```diff
@@ -196,58 +183,44 @@ function handleBehavior(feature: analyzer.PolymerBehavior, root: ts.Document) {
     return;
   }
   const [namespacePath, className] = splitReference(feature.className);
-  const parent = findOrCreateNamespace(root, namespacePath);
-  parent.members.push({
-    kind: 'interface',
-    name: className,
-    description: feature.description,
-    extends: [],
-    properties: handleProperties(feature.properties.values()),
-    methods: handleMethods(feature.methods.values()),
-  });
+  const i = new ts.Interface(className);
```

Comment:
I find the declarative style used before a bit easier to read. I if you need classes, you can have a constructor that takes a data object.

Reply:
Using data objects for most classes now.